server/athenian/api/__init__.py

```diff
@@ -357,6 +358,8 @@ def compose_db_options(mdb: str, sdb: str, pdb: str, rdb: str) -> Dict[str, Dict
 
 def main() -> Optional[AthenianApp]:
     """Server's entry point."""
+    from athenian.api.preloading.cache import MemoryCachePreloader
```

Comment:
This must be a conflict of the imports? I'll move `main()` to a separate module.